app/gateway-oss/2.3.x/upgrading.md

```diff
@@ -77,58 +78,55 @@ repository contains [openresty-build-tools](https://github.com/Kong/kong-build-t
 which allows you to build OpenResty with the necessary patches
 and modules easily.
 
-For Go support, you also need to build both your plugins and
-the [Kong go-pluginserver](https://github.com/kong/go-pluginserver).
-The documentation includes detailed [instructions on how to build
-the plugin server and your plugins](https://docs.konghq.com/2.2.x/go/).
+There is a new way to deploy Go using Plugin Servers. You can read more about it the [Go section of the docs](https://docs.konghq.com/2.3.x/go).
 
-#### 2. Template Changes
+### Template changes
 
 There are **Changes in the Nginx configuration file**, between kong 2.0.0,
-2.1.0 and 2.2.0.
+2.1.0, 2.2.0 and 2.3.0.
 
 To view the configuration changes between versions, clone the
 [Kong repository](https://github.com/kong/kong) and run `git diff`
 on the configuration templates, using `-w` for greater readability.
 
-Here's how to see the differences between 2.0.0 and 2.2.0:
+Here's how to see the differences between 2.0.0 and 2.3.0:
```

Comment:
Why is this diff between 2.0 and 2.3? shouldn't it be 2.2 and 2.3? 

This section is not in EE docs, so I'm not sure.

Reply:
It seems clear that the purpose of this section is not clear enough.

Upgrading within a mayor version (2.x to 2.y, where y > x) should be doable by only doing `kong migrations up`.

People who use a custom Nginx template should also do the extra step and check the changes in the template. They could be coming from 2.0.0, 2.1.x or 2.2.x. The examples use 2.0.0 for this reason. Perhaps we should add some phrase like "adjust the version number to the one you presently have".

Or perhaps we could set up an environmental variable, like this:

```
export UPGRADING_FROM=2.0.0
git diff -w "$UPGRADING_FROM" 2.2.0 kong/templates/nginx_kong*.lua
```
And then add "Please set `UPGRADING_FROM` to the Kong you are upgrading from".

The ideal solution would be for us to provide direct links on Github for the diffs of the individual file across versions:

* [Difference between 2.0.0 and 2.3.0](https://github.com/Kong/kong/compare/2.0.0...2.3.0#diff-d2fc92c2f1cd1ea7d01c13c68b069cbe9927088d2040d5c096906a93cf95bbc5)
* [Difference between 2.1.0 and 2.3.0](https://github.com/Kong/kong/compare/2.1.0...2.3.0#diff-d2fc92c2f1cd1ea7d01c13c68b069cbe9927088d2040d5c096906a93cf95bbc5)
* ...

Unfortunately those links are broken on the current github UI; the link provided on the file shows the whole diff.